modules/operator-upgrade-modify-central-custom-resource.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_module-type: PROCEDURE
+[id="operator-upgrade-modify-central-custom-resource_{context}"]
+= Modifying Central custom resource
+
+[role="_abstract"]
+The Central DB service requires persistent storage. If you have not configured a default storage class for {product-title-short} that is an SSD or is high performance, you must update the `Central` custom resource to request persistent storage for the Central DB service by creating a persistent volume claim (PVC).
```

Comment:
Once again I'm iterating over this as I try instructions. I noticed that this does **not** **create** a PVC, it **configures** the PVC. Specifically, it configures the storage class for the future PVC, however the PVC does not get created until the ACS is upgraded to 4.0 (or `isEnabled` is changed to `Enabled` which users must not do).

```suggestion
The Central DB service requires persistent storage. If you have not configured a default storage class for the cluster that is an SSD or is high performance, you must update the `Central` custom resource to configure the storage class for the Central DB persistent volume claim (PVC).
```